Programs/_freeze_module.c

```diff
@@ -158,7 +158,11 @@ write_code(FILE *outfile, PyObject *marshalled, const char *varname)
         }
         fprintf(outfile, "\n");
     }
-    fprintf(outfile, "};\n");
+    fprintf(outfile, "};\n\n");
+    fprintf(outfile, "const unsigned char *%s_ptr = %s;\n",
+        varname, varname);
+    fprintf(outfile, "const int %s_len = (int)sizeof(%s);\n",
+        varname, varname);
```

Comment:
What are these lines for?

Reply:
To make the values referenceable with an `extern` definition in another file (in this case, getpath.c) - we could equally put just the getpath values into `frozen.c` and have them reference the include file.

Note that it shouldn't make them (any more) visible to anyone outside of our build process.

Reply:
In getpath.c, why not use `_Py_M__getpath` directly?  That is effectively what frozen.c does.  It seems strange to add these two variables to each of the frozen files but only use them for one of them.  We could use all of them in frozen.c, but I don't see that adding much value.

All that said, it isn't a big deal either way.  I just favor the status quo. 🙂

Reply:
You can't read the length of it via an extern reference, because it's not a complete type. I _think_ I can reuse the pointer type, but possibly only by changing it for all uses.

Initially I did use them through all the rest of frozen.c, but yeah, doesn't add much. If we drastically grow the number of frozen modules we may reach a point where compiling all the data in a single .o file starts hitting size limits on some platforms. Then we can give them all their own .o and let the linker handle it.

I prefer status quo where it works - but it didn't work here ;)